### DISCUSSION TITLE
Arduino Amiga Floppy Disk support

### DISCUSSION THREAD

**Discussion by MaddTheSane** [dirkwhoffmann/vAmiga#527] [Ideas]
[Rob Smith](https://amiga.robsmithdev.co.uk) (@RobSmithDev) is working on making a USB floppy drive that interfaces with emulators and can run floppies directly from the floppy drive. He implemented the changes in his [WinUAE fork](https://github.com/RobSmithDev/WinUAE).

**Comment by dirkwhoffmann** [comment]
Cool project, I've watched a Youtube video about it. Although I do own original Amigas, I don't think that I can implement something similar in vAmiga in a reasonable amount of time (also, I have replaced the floppy drives in all of my Amigas by Gotek drives).

I'll move this topic to the discussion section. Maybe someone is eager to implement such a feature in vAmiga.
